modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java

```diff
@@ -182,18 +192,32 @@ protected void doServeResource(
 		}
 	}
 
-	private JSONArray _getAssetCategoriesJSONArray(
-		ContentDashboardItem contentDashboardItem, Locale locale) {
+	private HashMap<String, Object> _buildVocabularyDetails(
+			Locale locale, AssetCategory assetCategory,
+			AssetVocabulary currentVocabulary)
```

Comment:
As a Liferay style rule, we usually call the variable currentVocabulary with the same name that its type, so it should be renamed to assetVocabulary.